### PR TITLE
Bug 1189583 - [Nexus 4] Add support for the blobfree build

### PR DESCRIPTION
**Pull request by hobinjk**
Add TARGET_DEVICE_BLOBS and devices.json

### CODE REVIEW

File: devices.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+[
+  {
+    "id": "Nexus 4",
+    "adb": {
+      "ro.product.device": [ "mako" ],
+      "ro.bootloader": "MAKOZ20i",
+      "ro.build.id": "JSS15J"
+    },
+    "fastboot": {
+      "product": [ "mako" ]
```
Comment (lissyx) [inline devices.json:10]: No way we can check the bootloader version ? I'm pretty sure KK and L might differ.

File: full_mako.mk [hunk 2]
```diff
@@ -30,6 +30,11 @@ PRODUCT_MODEL := AOSP on Mako
 PRODUCT_MANUFACTURER := LGE
 PRODUCT_RESTRICT_VENDOR_FILES := true
 
+TARGET_DEVICE_BLOBS := vendor/lge/mako/device-partial.mk \
+                       vendor/broadcom/mako/device-partial.mk \
+                       vendor/qcom/mako/device-partial.mk
+
```
Comment (lissyx) [inline full_mako.mk:43]: Useless extra line

File: devices.json (hunk 1 above)
Comment (hobinjk) [inline devices.json:10]: I didn't get a result for `fastboot getvar version`

**Comment by hobinjk** [comment]
Accidentally messed up history by updating to track nexus-4-kk instead of nexus-4

**Pull request closed without merging**